unpackaged/config/trial/profiles/Admin.profile

```diff
@@ -100,6 +100,17 @@
         <layout>hed__Attribute__c-%%%NAMESPACE%%%K12 Kit Student Characteristic Layout</layout>
         <recordType>hed__Attribute__c.hed__Student_Characteristic</recordType>
     </layoutAssignments>
+    <layoutAssignments>
+        <layout>hed__Attribute__c-hed__EDA Credential Layout</layout>
+        <recordType>hed__Attribute__c.hed__Credential</recordType>
+    </layoutAssignments>
```

Comment:
This layout assignment is a duplicate of the K12 kit one. Let's use the K12 kit one for now.